pages/linux/paccache.md

```diff
@@ -0,0 +1,19 @@
+# paccache
+
+> A flexible pacman cache cleaning utility.
+
+- Remove all but the 3 most recent package versions from the pacman cache:
+
+`paccache -r`
```

Comment:
Is there a long form of this option? We generally prefer the longer form of options here at tldr-pages, as they are more descriptive for new users.

The same goes for the arguments in subsequent examples.

Reply:
I thought  that according to point 3 of the contribution guidelines it is enough to mention the long version of the option in the description of the command, since neither the pages for pacman nor pacaur use the long version of their options.
But if you prefer the long version, that's fine by me. Should I also change the other options?
-r -> --remove, -k -> --keep, -m -> --move, -d -> --dryrun

Reply:
Might be a good idea. Though _pacman_-based commands appear to be somewhat of an exception.

What do you suggest, @agnivade / @pxgamer?

Reply:
If the short-form option is unambiguously clear, which in this case, I think it is. Then it is fine to keep it. If `-r` meant something like update packages, then sure. Here since `-r` means `remove`, it is fine to use the short-form.

Reply:
I normally use long forms for most examples, but I think it seems fine using `-r` for `remove` here. 👍